pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -213,11 +214,14 @@ func (c *Controller) syncOwnedProvisionRequest(ctx context.Context, wl *kueue.Wo
 		attempt := int32(1)
 		shouldCreatePr := false
 		if exists {
-			if isFailed(oldPr) {
+			switch {
```

Comment:
using a switch for one case is not very standard. Consider using just a conditional and functions if nesting becomes too deep.

Reply:
Fixed